### PR TITLE
[DOCS-6082] query accelerator

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
HI Seeing if these are ok

### CODE REVIEW

File: query-accelerator/latest/config/index.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+}
+```
+
+## Query Set Status and Caching
+
+Denormalized tables have a status. For example:
+
+| Name | Version | State | Notes |
+| ---- | ------- | ----- | ----- |
+| tableA | 1 | OBSOLETE | About to be removed |
```
Comment (SaraAspery) [inline query-accelerator/latest/config/index.md:168]: "About to be removed" should be replaced with "Can be removed" or similar.
Also need an extra row in here:
| tableA | 2 | RETIRED | Previous LIVE version. Can be removed |

File: query-accelerator/latest/config/index.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+| tableA | 2 |  LIVE | Currently being used |
+| tableA | 3 |  INPROGRESS | Created but not fully populated yet, so cannot be used |
+| tableA | 4 |  NEW | Seen but population of denormalized data has not started |
```
Comment (SaraAspery) [inline query-accelerator/latest/config/index.md:171]: Should be version 5

File: query-accelerator/latest/config/index.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+## Query Set Status and Caching
+
+Denormalized tables have a status. For example:
+
+| Name | Version | State | Notes |
+| ---- | ------- | ----- | ----- |
+| tableA | 1 | OBSOLETE | About to be removed |
+| tableA | 2 |  LIVE | Currently being used |
+| tableA | 3 |  INPROGRESS | Created but not fully populated yet, so cannot be used |
```
Comment (SaraAspery) [inline query-accelerator/latest/config/index.md:170]: Should be version 4

File: query-accelerator/latest/config/index.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+```
+
+## Query Set Status and Caching
+
+Denormalized tables have a status. For example:
+
+| Name | Version | State | Notes |
+| ---- | ------- | ----- | ----- |
+| tableA | 1 | OBSOLETE | About to be removed |
+| tableA | 2 |  LIVE | Currently being used |
```
Comment (SaraAspery) [inline query-accelerator/latest/config/index.md:169]: Should be version 3

**Review by SaraAspery — CHANGES_REQUESTED** [review]
Looks good Oliver! Just need a table update. Sorry about that, the update is missing from the original docs.

File: query-accelerator/latest/config/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply query-accelerator/latest/config/index.md:168]: done

File: query-accelerator/latest/config/index.md (hunk 3 above)
Comment (oliveralfresco2018) [reply query-accelerator/latest/config/index.md:170]: Changed | tableA | 3  to be | tableA | 4

File: query-accelerator/latest/config/index.md (hunk 4 above)
Comment (oliveralfresco2018) [reply query-accelerator/latest/config/index.md:169]: changed | tableA | 2 to be | tableA | 3

File: query-accelerator/latest/config/index.md (hunk 2 above)
Comment (oliveralfresco2018) [reply query-accelerator/latest/config/index.md:171]: Changed | tableA | 4 to | tableA | 5

**Comment by oliveralfresco2018** [comment]
Change the version in the drop down to be 7.0 this will always reflect the ACS version.

**Comment by oliveralfresco2018** [comment]
I have made all the changes to these docs and they are good to be released. 
They will have more work and polishing on them after we go live.

**Review by SaraAspery — APPROVED** [review]

**Review by SaraAspery — APPROVED** [review]

**Comment by SaraAspery** [comment]
Well spotted Oliver!

File: query-accelerator/latest/index.md [hunk 5]
```diff
@@ -26,17 +26,15 @@ be possible to replace a query set with a new version or to remove it completely
 properties or aspects applied to nodes and if necessary (for selected databases) the order of columns
 in compound indexes. Query sets are defined using JSON files.
 
-3. Administrator to perform a query set refresh in the Alfresco Administration Console. The addition of new query sets, 
-the replacement of an existing query set or complete removal does not require a 
-restart, an outage or have a major impact on normal operations. The alfresco.log will contain messages to reflect 
-progress. When a new query set is identified, the system will start populating a denormalized 
-table in background. It will also abandon the table population before it is complete, if a new 
+3. Administrator's perform a query set refresh in the Alfresco Administration Console. The addition of new query sets,
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply query-accelerator/latest/index.md:29]: rectified

**Review by gravitonian — APPROVED** [review]
Looks OK

**Pull request merged**